src/output.rs

```diff
@@ -524,7 +532,6 @@ pub fn output_html(
     } else {
         PathBuf::from("./html")
     };
-
```

Comment:
Nit: unrelated newline removal, can you add it back?

Reply:
locally, i have a new line in this file

Reply:
pushed